atlas-fastq-provider-functions.sh

```diff
@@ -370,14 +370,25 @@ fetch_file_by_wget() {
             wait_and_record ${source}_$method
         fi
         wget  -nv -c $sourceFile -O $wgetTempFile 
+        wget_status=$?
+        
+        # If this is a gzip file, we can test its integrity
+
+        gzip_status=0
+        echo "$sourceFile" | grep '\.gz$' > /dev/null   
```

Comment:
I am wondering if you should do an additional check on the `$wgetTempFile`, in addition to $sourceFile. 
For instance if its size if > 0 or similar, but I think this can be captured by the gzip command below

Reply:
Sure, I'll add a file size check too (not everything will necessarily be gzipped). Thanks!

Reply:
Actually, no need, there's already a '-s' check just below.